packages/components/checkboxes/template.njk

```diff
@@ -12,39 +12,39 @@
    aria-describedby – for example hints or error messages -#}
 {% set describedBy = "" %}
 
-{% set isConditional = false %}
-{% for item in params.items %}
-  {% if item.conditional %}
-    {% set isConditional = true %}
-  {% endif %}
-{% endfor %}
+{#- Jinja compatability - avoid referencing params.items/params.values directly
```

Comment:
I had another idea for dealing with the name collision here - we could transform it within the macro.njk, so that the templates themselves use a different name (e.g. `checkoxItems`) but the external API remains the same.